addons/base_vat/migrations/11.0.1.0/openupgrade_analysis.txt

```diff
@@ -3,3 +3,4 @@
 NEW ir.ui.view: base_vat.res_config_settings_view_form
 NEW ir.ui.view: base_vat.view_partner_short_form
 DEL ir.ui.view: base_vat.company_form_vat
+# Nothing to do
```

Comment:
You should create a new `openupgrade_analysis_work.txt` file, not modifying this one.